capitalizeDecorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'baseDecorator'
+
+class CapitalizeDecorator < BaseDecorator
+  def correct_name
+    puts @nameable.upcase
+  end
```

Comment:
- [ ] This method should capitalize only the first letter of the given name not the whole name, kindly consider fixing this so it returns the correct value